docs/assets/index-DmJnWrCa.js

```
@@ -0,0 +1,84 @@
+      const n = await window.kasware.getKRC20Balance();
+      console.log('[DEBUG] KRC20 balance response:', n);
+      const t = n.find((r) => r.tick.toUpperCase() === a.toUpperCase());
+      if ((console.log('[DEBUG] Found tick balance:', t), t)) {
+        const r = Number(t.dec);
+        return console.log('[DEBUG] Got decimals from wallet:', r), r;
+      } else {
+        const r = e[a.toUpperCase()] || 8;
+        return console.log('[DEBUG] Token not found in wallet, using fallback decimals:', r), r;
+      }
+    } catch (n) {
+      console.warn('Failed to get KRC20 decimals:', n), console.error('[DEBUG] Error details:', n);
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)